zipkin-ui/test/component_ui/traceToMustache.test.js

```diff
@@ -40,20 +40,21 @@ const span3 = span(12345, 'methodcall2', span3Id, span2Id, 300, 50, annotations3
 const span4 = span(12345, 'methodcall2', span4Id, span3Id, 400, 100, annotations4);
 
 const trace = [span1, span2, span3, span4];
+const logsUrl = '';
```

Comment:
I would make this method scope, and set it to undefined as I think that's what it would be in practice.
then, add a test for what happens when it isn't undefined. For example, traceToMustache result includes the logsUrl